config/default/common/config/metadata/stories/fires_overview_2019/step001.md

```diff
@@ -0,0 +1,6 @@
+<p>NASA provides insights into fires and thermal anomalies occurring daily around the world. Fire information available in Worldview are from the MODIS instrument on board the Terra and Aqua satellites and the VIIRS instrument on board the joint NASA/NOAA Suomi-NPP satellite. Terra passes over the equator at approximately 10:30am (Day) and 10:30pm (Night) local time and Aqua and Suomi-NPP passes over the equator at approximately 1:30pm (Day) and 1:30am (Night) local time. The fire information are available within Worldview approximately 3 hours after satellite overpass.
```

Comment:
"Fire information available in Worldview are from" -> "Fire information available in Worldview **_is_** from"

Reply:
reworded it